examples/cordova/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "com.metriport.cordova",
+  "displayName": "MetriportTestApp",
+  "version": "1.0.0",
+  "description": "A sample Metriport Cordova application that uses our healthkit plugin.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
nit: maybe just leave this out, since its an example and the test script doesn't add to it?